src/api/spec/controllers/webui/cloud/azure/upload_jobs_controller_spec.rb

```diff
@@ -0,0 +1,230 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Webui::Cloud::Azure::UploadJobsController, type: :controller, vcr: true do
+  let(:azure_configuration) { build(:azure_configuration, application_id: 'Hey OBS!', application_key: 'Hey OBS?') }
+  let(:user_with_azure_configuration) { create(:confirmed_user, login: 'tom', azure_configuration: azure_configuration) }
+  let(:project) { create(:project, name: 'Apache') }
+  let!(:package) { create(:package, name: 'apache2', project: project) }
+  let(:upload_job) { create(:upload_job, user: user_with_azure_configuration) }
+  let(:xml_response) do
+    <<-HEREDOC
+    <clouduploadjob name="6">
+      <state>created</state>
+      <details>waiting to receive image</details>
+      <created>1513604055</created>
+      <user>mlschroe</user>
+      <target>azure</target>
+      <project>Base:System</project>
+      <repository>openSUSE_Factory</repository>
+      <package>rpm</package>
+      <arch>x86_64</arch>
+      <filename>rpm-4.14.0-504.2.x86_64.rpm</filename>
+      <size>1690860</size>
+    </clouduploadjob>
+    HEREDOC
+  end
+  let(:public_key) { File.read(Rails.root.join('spec', 'support', 'files', 'cloudupload_public_key.txt')) }
```

Comment:
No need to construct this manually, RSpec knows [file_fixture](https://relishapp.com/rspec/rspec-rails/docs/file-fixture)

Reply:
Yes, I was following what was done in the Cloud::Azure::Configuration time ago... I will change both